### PR TITLE
[feature]: add bitte-wallet package

### PR DESCRIPTION
**Pull request by sainthiago**

### CODE REVIEW

File: packages/bitte-wallet/README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+check our [React integration](https://docs.mintbase.xyz/dev/mintbase-sdk-ref/react#mintbasewalletcontextprovider)
+
+Example:
+You can check a [quick example of Simple Login](https://github.com/Mintbase/examples/tree/main/starter) using Next.js 14 and @mintbase-js/react
+
+
+<p align="center">
+<img  src='https://img.shields.io/npm/dw/@mintbase-js/wallet'  />
+<img  src='https://img.shields.io/bundlephobia/min/@mintbase-js/wallet'>
```
Comment (rubenmarcus) [inline packages/bitte-wallet/README.md]: `@mintbase-js/bitte-wallet`

File: packages/bitte-wallet/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+const  bitteWalletØ = setupBitteWallet({
+    networkId:  'mainnet',
+    walletUrl:  'https://wallet.bitte.ai',
+    callbackUrl:  'https://www.mywebsite.com',
+    deprecated:  false,
+});
+
+const  selector = await  setupWalletSelector({
+    network:  "mainnet",
+    modules: [bitteWalletØ],
```
Comment (rubenmarcus) [inline packages/bitte-wallet/README.md:115]: `bitteWallet`

File: packages/bitte-wallet/README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+{% code title="setup.ts" overflow="wrap" lineNumbers="true" %}
+
+
+
+
+```ts
+import { setupWalletSelector } from  "@near-wallet-selector/core";
+import { setupBitteWallet } from  "@mintbase-js/bitte-wallet";
+
+const  bitteWalletØ = setupBitteWallet({
```
Comment (rubenmarcus) [inline packages/bitte-wallet/README.md]: `bitteWallet`

File: packages/bitte-wallet/README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    modules: [bitteWalletØ],
+});
+```
+
+{% endcode %}
+
+
+
+
+## setupMintbaseWallet
```
Comment (rubenmarcus) [inline packages/bitte-wallet/README.md:124]: `## setupBitteWallet
`

File: packages/bitte-wallet/src/setup.ts [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
...sovV9cSlaWMBlOUHFmjfMUpgFOGt266WH+Sg7JjiUjQejJ5dxCFLwi4xyyLVdupe8td1wuoMkp3HIrB3UsGQjCaCfOvJnwZPB5HZzLoBeK8+K0fZrODRP6a+l2TwLnTVwXOo/L4XPag2iCYLFJR4tr9+Fcx/vvQwApC8LndfDEmyYyLq1HJ4dbLNBPxfXXuUi+C4sRhiAkgSLSpRLAQJq77rLXYgzgHXzvvfYdw2eJZDQLmICU59R51ZticCqUYQZRZTSnTO+7r/WKos3/rYTXCB0ophg0UdfXR4tBuECm6+mi10oA6QrCIsGE8Rs88ZVgvB6DKk2S1h9cjN+EVzptPJVgMI45G8giT8AjK9IV5Mcfw9f0zMaZ/qRgjLloD1xt1KRSfEcARefTdQJ4V1ZzS3YA6Qpinpgk9spyT7+Jy6xZdjrusekgMSnc4ypMJ3mShe3nzrXTNDUkYfXVK39f3OOusQbyJl1BTCsnrTtx4SOJTExHehzcuI647nkz8olFWNz4kAxJVxA38JFLsVaDwjHCNaAtTvIjj7TT9TEefM0murtoP1fBzpl0BWHHz+SBB4C//iq/P08K2//mag1J7w5y8MF2McfO36uvlt+fFTj7KOZAJ+9uM24wJ9IV5JBDbFchI6No92dfhCfB3Nhb5vIUZq+Znl7GfbeFK68MX/P7b7lFhw3w2VTmcdkAYDy8+QAwFlftMXunSPpDJ6zMGezvLlLM4ExegeuuC0ye3LrS50mh3T/pkIkJn2NoFn8BrBu4di+HatjfcI971llSnpaQ0eAiV5LmUEbcJ91wmINBMr16oV1wOIaicDwtDhSDS3kwCEcGGYUjcLCOznRGQVXqdfM9VqQc9mivGISdTAbxV5vn4Hu8U7lyhBwxmsk+YIdNWq5fwnWp2JOfN09PELFFxvrGbSGlBe9SLsrPJ0VzfI1FFdfD4jA7V4DgIGi1YZ2Op2APJ/7/U7CHExcACtIIjxQaKEgDPFKYSEFGwSMB1uVvslLnooE0RnWFJ08mlEql7p3UP/+oRH948oR3Rx1fNLeylCh0GgyBJw8oxmClwbNMWN1ZVXzVqT8D3XxPZlCMoUqM64KMVideiVILLUoNvDBZQSE4XdlfiTHCfKPsCW8Rpg+0MG1c1cXj0AjdzRiptmdb6m+LfwEGoUIy8K/6BwAAAABJRU5ErkJggg==';
```
Comment (rubenmarcus) [inline packages/bitte-wallet/src/setup.ts:21]: add svg here

File: packages/bitte-wallet/src/utils.ts [hunk 6]
```diff
@@ -0,0 +1,84 @@
+/*
+    Mintbase Wallet Utils file
```
Comment (rubenmarcus) [inline packages/bitte-wallet/src/utils.ts:2]: Bitte Wallet

**Review by rubenmarcus — CHANGES_REQUESTED** [review]

File: packages/bitte-wallet/README.md (hunk 1 above)
Comment (sainthiago) [reply packages/bitte-wallet/README.md]: fixed

File: packages/bitte-wallet/README.md (hunk 3 above)
Comment (sainthiago) [reply packages/bitte-wallet/README.md]: fixed

**Review by sainthiago — COMMENTED** [review]
closed for #517

**Pull request closed without merging**